code/game/objects/items/trash.dm

```diff
@@ -84,8 +95,24 @@
 	icon_state = "liquidfood"
 
 /obj/item/trash/chicken_bucket
-	name = "Chicken bucket"
+	name = "chicken bucket"
 	icon_state = "kfc_bucket"
 
-/obj/item/trash/attack(mob/M as mob, mob/living/user as mob)
-	return
+//Special behavior below if warranted
+
+//Is someone being a cunt at the table again ?
+/obj/item/trash/plate/attack(mob/M as mob, mob/living/user as mob)
+	user.delayNextAttack(10)
+	user.visible_message("<span class='danger'>[user] shatters \the [src] on [M]'s head</span>", "<span class='warning'>You shatter \the [src] on [M]'s head</span>")
+	if(isliving(M))
+		var/mob/living/L = M
+		L.apply_damage(20, BRUTE, "head")
+	playsound(get_turf(src), "shatter", 70, 1)
+	if(ishuman(M))
+		var/mob/living/carbon/H = M
+		H.Stun(3)
+		H.emote("scream",,, 1)
+	for(var/mob/O in viewers(M, null))
+		shake_camera(O, 2, 3)
+	user.drop_item(src)

```

Comment:
The arg for drop_item is not the item to drop.
It's the place to put it. I'm genuinely surprised this didn't make a runtime when you placed the plate in itself.